receiver/vcenterreceiver/testdata/metrics/expected.json

```diff
@@ -1,4000 +1,4029 @@
 {
-    "resourceMetrics": [
+  "resourceMetrics": [
```

Comment:
It seems this PR is changing the indentation of this file, which is making it difficult to understand the diff. Do you mind seeing if you can commit this without the indentation change?

Reply:
Yes,  that would be a lot easier 🤦‍♂️